app/modules/asset_groups/models.py

```diff
@@ -457,7 +466,7 @@ def detected(self, job_id, response):
 
             for annot_id in range(len(results)):
                 annot_data = results[annot_id]
-                annot_uuid = annot_data.get('uuid', {}).get('__UUID__')
```

Comment:
yup i am going to say that sage _is_ still formatting uuids wrapped in `{ "__UUID__": value }` so i think this needs to be back in here.